src/coffee/product-type-generator.coffee

```diff
@@ -196,8 +196,22 @@ class ProductTypeGenerator
     i18n = {}
     languages = @_languages header, _.keys row
     for language in languages
-      if row["#{header}.#{language}"].trim() isnt ''
-        i18n[language] = row["#{header}.#{language}"].trim()
+      # condition was commented out due to an error:
```

Comment:
There was one issue that when you have a productType with `LENUM` attribute it exported first empty value like this:
```
"values": [
{
  "key": "",
  "label": {}
}
```
Which is wrong because you are missing a value in label and Sphere thus you will get an error: `attributes -> type -> elementType -> values -> label: Values of LocalizedString must not be empty.`
So I removed that `if` and now creates first empty value with language code:
```
"label": {
    "de": ""
}
```

Anyway - I removed that comment.